service/src/main/java/tds/assessment/services/impl/AssessmentSegmentLoaderServiceImpl.java

```diff
@@ -175,7 +175,7 @@ public void loadAdminSubjects(final TestPackage testPackage, final String subjec
                         .withFieldTestMinItems(segmentBpElement.minFieldTestItems())
                         .withFieldTestMaxItems(segmentBpElement.maxFieldTestItems())
                         .withSelectionAlgorithm(segment.getAlgorithmType().equals("adaptive") ? Algorithm.ADAPTIVE_2.getType() : segment.getAlgorithmType())
-                        .withBlueprintWeight(Float.parseFloat(itemSelectionProperties.getOrDefault("blueprintweight", DEFAULT_BLUEPRINT_WEIGHT)))
+                        .withBlueprintWeight(Float.parseFloat(itemSelectionProperties.getOrDefault("bpweight", DEFAULT_BLUEPRINT_WEIGHT)))
```

Comment:
I suppose another fix here would be to look for either `bpweight` or `blueprintweight`.  Unfortunately the XSD can't help us out here much; the `<itemSelection>` element can have any string for its `name` attribute.  But from what I've gathered, the convention for blueprint weight is `bpweight`.  Hence the change.